Automattic/wp-calypso
# Subscriptions: Render the "Preview and edit" link for the Subscription block after the post toggle

Related to peKye1-EA-p2

## Proposed Changes

It renders the "Preview and edit" link for the Subscription block after the post toggle for block themes.

<img width="737" alt="Screenshot 2024-04-29 at 14 30 41" src="https://github.com/Automattic/wp-calypso/assets/4068554/ffab7538-446e-4393-956d-0b02ac0e123c">
<br><br>

Blocked by https://github.com/Automattic/jetpack/pull/37123

## Testing Instructions

1. Go to Newsletter Settings
2. Enable the "Add the Subscribe Block at the end of each post" toggle
3. Make sure the "Preview and edit" link next to it takes you to the Single Posts template editor where you are able to edit the Subscription block

## Pre-merge Checklist
